apps/code-examples/src/app/code-examples/indicators/icon/basic/icon-demo.component.html

```diff
@@ -0,0 +1,26 @@
+<sky-icon icon="calendar"></sky-icon>
+<sky-icon icon="calendar" size="lg"></sky-icon>
+<sky-icon icon="calendar" size="2x" iconType="skyux"></sky-icon>
+<sky-icon icon="calendar" iconType="skyux" variant="line" size="3x"></sky-icon>
+<sky-icon
+  data-sky-id="icon-demo"
+  icon="calendar"
+  iconType="skyux"
+  variant="solid"
+  size="4x"
+  [fixedWidth]="true"
+></sky-icon>
+
+<div>
```

Comment:
added these to show what fixedWidth does

Reply:
Nice demo.